_posts/2023-09-28-remote-backed-storage.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title:  "Improved durability in OpenSearch with remote-backed storage"
+authors:
+- kalsac
+- gbbafna
+- ssashish
+- rohin
+- bukhtawa
+- ramaran
+- nbower
+date: 2023-09-27 00:00:00 -0700
+categories:
+ - community
+
```

Comment:
Add meta: 
**meta_keywords:** OpenSearch remote backed storage, configured remote store, achieve a recovery point objective

**meta_description:** Learn how the OpenSearch remote-baked storage features helps users achieve a recovery point objective of zero while leveraging the durability properties of a configured remote store.